grails-app/controllers/org/pih/warehouse/putaway/PutAwayController.groovy

```diff
@@ -23,7 +23,7 @@ class PutAwayController {
     // This template is generated by webpack during application start
     def create() {
         def contextPath = grailsApplication.config.server.contextPath
-        render(template: "/common/react", model: [contextPath: contextPath != '/' ? contextPath : ''])
+        render(view: "/common/react", model: [contextPath: contextPath != '/' ? contextPath : ''])
```

Comment:
I think you can safely remove the `model` part here since the context part is taken from util inside the template.

Reply:
removed. Working fine